client/src/components/Header/components/universal-nav.css

```
@@ -391,23 +387,20 @@ li > button.nav-link-signout:not([aria-disabled='true']):is(:hover, :focus) {
     margin-top: 0;
   }
 
-  .display-menu {
-    top: calc(var(--header-height) * 2);
-  }
-
   .fcc_searchBar,
   .fcc_searchBar div {
     width: 100%;
   }
 
-  .universal-nav-right .fcc_searchBar .ais-SearchBox-form {
-    width: calc(100% - 30px);
-  }
-
   .display-menu {
+    top: calc(var(--header-height) * 2);
     max-height: calc(100vh - var(--header-height) * 2);
   }
 
+  .universal-nav-right .fcc_searchBar .ais-SearchBox-form {
+    width: calc(100% - 30px);
+  }
+
   .universal-nav-right .fcc_searchBar .ais-Hits {
     width: calc(100% - 30px);
   }
```

A small nitpick, same property and value.
```suggestion
  .universal-nav-right .fcc_searchBar .ais-Hits,
  .universal-nav-right .fcc_searchBar .ais-SearchBox-form {
    width: calc(100% - 30px);
  }
```

---

Thanks for your review.
And sorry, I overlooked that they have same property and value so  putting together them will make the code  readabile and clean.